recipe/meta.yaml

```diff
@@ -20,7 +20,9 @@ requirements:
     - pip
     - setuptools
   run:
-    - python >={{ python_min }}
+    # excluding python 3.14 for now because we don't have the required grpcio on conda-forge
+    - python >={{ python_min }},<3.14
+    # just for python <3.14 for now
```

Comment:
I'm not seeing movement on grpcio 1.75.1 or 1.76.0 on conda-forge (which is understandable -- it's a beast!). I think the way forward for `google-cloud-*` for now has to be excluding python 3.14.